eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -14,31 +14,35 @@ jobs:
       # TRANSPORT_PAHO              OFF
       # PRECONDITIONS               ON
       # LOGGING                     ON
-      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_NONE
+      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_POSIX / AZ_PLATFORM_IMPL_WIN32
```

nit: kinda looks like division
```suggestion
      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_POSIX | AZ_PLATFORM_IMPL_WIN32
```